python/ray/dag/compiled_dag_node.py

```diff
@@ -798,24 +882,17 @@ def execute(
 
         Args:
             args: Args to the InputNode.
-            kwargs: Kwargs to the InputNode. Not supported yet.
+            kwargs: Kwargs to the InputNode
 
         Returns:
             A list of Channels that can be used to read the DAG result.
         """
-        # These errors should already be caught during compilation, but just in
-        # case.
-        if len(args) != 1:
-            raise NotImplementedError("Compiled DAGs support exactly one InputNode arg")
-        if len(kwargs) != 0:
-            raise NotImplementedError("Compiled DAGs do not support kwargs")
-
         if self._enable_asyncio:
             raise ValueError("Use execute_async if enable_asyncio=True")
 
         self._get_or_compile()
 
-        inp = args[0]
+        inp = (args, kwargs)
         if self.input_wrapper_fn is not None:
```

Comment:
TODO: double check this (TorchTensorWrapper) works properly.

Reply:
Ah I believe this actually should've been removed in #45092, feel free to remove it.

Reply:
Thanks. Removed.